src/services/todos.ts

```diff
@@ -0,0 +1,16 @@
+import todosFromServer from '../api/todos';
+import { Todo } from '../types/Todo';
+import { getUserById } from './user';
+
+export const filteredTodos = todosFromServer
+  .map(todo => ({
+    ...todo,
+    user: getUserById(todo.userId),
+  }))
+  .filter(todo => todo.user?.id);
```

Comment:
why do you need this filter?
you can just handle case when user is undefined

Reply:
I deleted this filter.
In my opinion, it is needed in order to discard standard titles in which there are no users. I know that there are none, but in theory they could be)